### DISCUSSION TITLE
Hope we can use  `LabelSizeFontToFit` in CommunityToolkit.Maui  just as   XamarinCommunityToolkit does.

### DISCUSSION THREAD

**Discussion by jessiezh0320** [CommunityToolkit/Maui#697] [New Feature Discussions]
We notice that we can use `LabelSizeFontToFit`  for Label  in  [XamarinCommunityToolkit](https://github.com/xamarin/XamarinCommunityToolkit),for example:

 ```
<Label x:Name="pbText" Text=" " HorizontalOptions="StartAndExpand" VerticalOptions="Center" 
	Font="{StaticResource fontAccNoCases}" LineBreakMode="NoWrap" >
        <Label.Effects>
               <effects:LabelSizeFontToFit />
         </Label.Effects>
</Label>
```
And I found  there is a relative issue about this here: https://github.com/xamarin/XamarinCommunityToolkit/issues/60 .

I hope we can also use the same feature in CommunityToolkit.Maui.

**Comment by VladislavAntonyuk** [comment]
It was a plan to add it to Xamarin: https://github.com/xamarin/Xamarin.Forms/pull/7981/files

**Reply by RobTF** [reply]
+ 1

**Comment by CliffAgius** [comment]
Closed as per the Community Stand-up discussion here: https://youtu.be/N9wMcBP4jtg?t=2889
